apps/cpp_rpc/rpc_env.cc

```diff
@@ -67,7 +67,12 @@ namespace {
 namespace tvm {
 namespace runtime {
 RPCEnv::RPCEnv() {
-  base_ = "./rpc";
+  char cwd[PATH_MAX];
+  if (char *rc = getcwd(cwd, sizeof(cwd))) {
+    base_ = std::string(cwd) + "/rpc";
+  } else {
+    base_ = "./rpc";
+  }
```

Comment:
This will break on Windows platform. Windows doesn't have `PATH_MAX` (is 4096) but has `MAX_PATH` (is 260). Please guard it on different macro. Or you could define one common length instead of using `PATH_MAX`. cc @jmorrill

Reply:
I don't have a convenient way of testing it on Windows right now, so I'm just going to #ifdef it.

Reply:
Ok. Could you triger CI again? The error shouldn’t be related with your change.

Reply:
Done.